.github/workflows/publish.yaml

```
@@ -0,0 +1,15 @@
+name: docker_publish
+
+on:
+  push:
+    branches:
+      - 'f/ci'
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  docker_pipeline:
+    uses: babylonlabs-io/.github/.github/workflows/reusable_docker_pipeline.yml@v0.3.1
```

```suggestion
    uses: babylonlabs-io/.github/.github/workflows/reusable_docker_pipeline.yml@v0.6.0
```

Bumping to latest version supporting multi-platform builds.